evm/src/cpu/columns.rs

```diff
@@ -153,4 +155,40 @@ pub const SIMPLE_LOGIC_OUTPUT: Range<usize> = SIMPLE_LOGIC_INPUT1.end..SIMPLE_LO
 pub const SIMPLE_LOGIC_DIFF: usize = SIMPLE_LOGIC_OUTPUT.end;
 pub const SIMPLE_LOGIC_DIFF_INV: usize = SIMPLE_LOGIC_DIFF + 1;
 
-pub const NUM_CPU_COLUMNS: usize = SIMPLE_LOGIC_DIFF_INV + 1;
+const NUM_MEMORY_OPS: usize = 4;
+const NUM_MEMORY_VALUE_LIMBS: usize = 8;
+
+pub(crate) const MEMORY_TIMESTAMP: usize = SIMPLE_LOGIC_DIFF_INV + 1;
```

Comment:
nit: I'd prefer to call it `CLOCK` to make it clear it's measured in cycles, and it might have other uses besides memory